src/main/java/org/myrobotlab/service/GoogleSearch.java

```diff
@@ -385,12 +384,12 @@ public static void main(String[] args) {
 
   @Override
   public String getLanguage() {
-    return locale.getLanguage();
+    return Runtime.getInstance().getLanguage();
   }
 
   @Override
   public Locale getLocale() {
-    return locale;
+    return Runtime.getInstance().getLocale();

```

Comment:
Should be locale of the google search service..  if that's null, then fallback to runtime locale.

Reply:
fixed - taken care of by parent Service